pkgs/development/libraries/libdrm/default.nix

```diff
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" "bin" ];
 
   nativeBuildInputs = [ pkgconfig meson ninja ];
-  buildInputs = [ libpthreadstubs libpciaccess valgrind-light ];
+  buildInputs = [ libpthreadstubs libpciaccess ]
+    ++ lib.optional valgrind-light.meta.available valgrind-light;
```

Comment:
Since this is an optional dependency, we should have a flag that is something like ```withValgrind```. Also the meta.available check is not so good because it also takes into account allowBroken. Instead, we should use ```lib.any (lib.meta.platformMatch stdenv.hostPlatform) valgrind-light.meta.platforms```

Reply:
That check does not work because `valgrind` uses `meta.badPlatforms`.